features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -595,6 +595,24 @@
 				<AttributeID>active</AttributeID>
 				<Description>Status of the account</Description>
 			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/secretkey</ClaimURI>
+				<DisplayName>Secret Key</DisplayName>
+				<AttributeID>totpSecretkey</AttributeID>
```

Comment:
If someone is using user store backed identity data store it would be better to add a note in the docs to add a valid mapped attribute?

Reply:
Will include in the docs